march_hardware_interface/src/march_hardware_interface_node.cpp

```diff
@@ -41,7 +41,7 @@ int main(int argc, char** argv)
     return 1;
   }
 
-  const double loop_hz = ros::param::param("~loop_hz", 100.0);
+  const double loop_hz = ros::param::param("~loop_hz", 250.0);
```

Comment:
Is this increased to the frequency of ethercat, because of the watchdog, because then this should forced to be equal by using the same parameter from the yaml?

Reply:
Yes it was increased to synchronize the controller and ethercat loop, see my other comment for deriving this from the ethercat cycle time.